### PR TITLE
feat: slicing layout filter component for master data publication

### PR DESCRIPTION
**Pull request by doohanas**
### Related
- [S55A11 - Filter Master Data Layanan](https://app.clickup.com/t/860rcqc7q)

### Changes
1. feat: slicing layout filter component for master data publication

### Preview

1. Filter with super admin role

https://github.com/jabardigitalservice/portal-jabar-cms/assets/79241754/c7b8b08f-58d8-4cf8-b62d-25c33d6ba1c4

2. Filter with non-super admin role

https://github.com/jabardigitalservice/portal-jabar-cms/assets/79241754/abb4bd84-699c-4163-955b-988fafd34cf9

### Evidence
title: feat: slicing layout filter component for master data publication
project: Portal Jabar
participants: @doohanas @Ibwedagama @naufalihsank @yoslie

### CODE REVIEW

File: src/components/Services/serviceFilter.vue [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+          class="w-full mb-6"
+        >
+          <div class="min-w-0 w-full flex gap-3 items-center mb-4">
+            <p class="font-lato text-sm text-blue-gray-700 leading-none whitespace-nowrap">
+              Author
+            </p>
+            <div class="w-full border border-gray-100" />
+          </div>
+          <JdsSelect
+            placeholder="Pilih Author"
```
Comment (naufalihsank) [inline src/components/Services/serviceFilter.vue:62]: minor

intentional mixing language ?

File: src/components/Services/serviceFilter.vue [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+                class="py-2"
+              >
+                <p class="text-[13px] text-red-500">
+                  Tanggal awal tidak boleh kosong
+                </p>
+              </div>
+            </div>
+            <div>
+              <JdsDateInput
+
```
Comment (naufalihsank) [inline src/components/Services/serviceFilter.vue:116]: minor

blank space

File: src/components/Services/serviceFilter.vue [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+              <div
+                v-show="false"
+                class="py-2"
+              >
+                <p class="text-[13px] text-red-500 leading-5">
+                  Tanggal akhir tidak boleh kurang
+                  <br>
+                  dari tanggal awal
+                </p>
+              </div>
```
Comment (naufalihsank) [inline src/components/Services/serviceFilter.vue:136]: are these `v-show` conditions will be created in separate PR ?

**Review by naufalihsank — APPROVED** [review]

File: src/components/Services/serviceFilter.vue (hunk 1 above)
Comment (doohanas) [reply src/components/Services/serviceFilter.vue:62]: what do you mean about the text in the placeholder?

File: src/components/Services/serviceFilter.vue (hunk 3 above)
Comment (doohanas) [reply src/components/Services/serviceFilter.vue:136]: Yes, that directive will be created in a separate PR.

File: src/components/Services/serviceFilter.vue (hunk 2 above)
Comment (doohanas) [reply src/components/Services/serviceFilter.vue]: change [here](https://github.com/jabardigitalservice/portal-jabar-cms/pull/455/commits/be3bf1105345056ea15b729c7c4face62da71c27)

File: src/components/Services/serviceFilter.vue (hunk 1 above)
Comment (naufalihsank) [reply src/components/Services/serviceFilter.vue:62]: my concern is just "Author" isn't indonesian language, both in input label (`p`) and the placeholder for `JDSSelect`

but if it is ok with PM, no problem then

**Review by Ibwedagama — APPROVED** [review]

**Review by yoslie — APPROVED** [review]

File: src/components/Services/serviceFilter.vue (hunk 1 above)
Comment (doohanas) [reply src/components/Services/serviceFilter.vue:62]: Yes, it comes from the design and it should be approved by PM.
